chord_metadata_service/chord/ingest/phenopackets.py

```diff
@@ -22,6 +24,16 @@
 from typing import Any, Optional, Union
 
 
+def _get_or_create_opt(key: str, data: dict, create_func):
```

Comment:
can type create_func and return value here using Callable and python's hack-y generic/typevar system

Reply:
Done, good idea